backend/src/test/java/ca/bc/gov/hlth/hnweb/controller/EligibilityControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package ca.bc.gov.hlth.hnweb.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.server.ResponseStatusException;
+
+import ca.bc.gov.hlth.hnweb.model.CheckEligibilityResponse;
+import ca.bc.gov.hlth.hnweb.service.EligibilityService;
+import ca.uhn.hl7v2.HL7Exception;
+
+@SpringBootTest
+public class EligibilityControllerTest {
+	
+	/**
+	 * Inject the service which needs to be mocked out. @MockBean tells Spring to mock out this bean
+	 * anytime it's injected
+	 */
+	@MockBean
+	private EligibilityService eligibilityServiceMock;
+
+	/**
+	 * Inject the Controller being tested using the standard @Autowired annotation.
+	 */
+	@Autowired
+	private EligibilityController eligibilityController;
+	
+	@Test
+	public void testCheckEligibility_success() throws HL7Exception, IOException, ParseException {
```

Comment:
HL7Exception and IOException are not thrown in this method

Reply:
Removed.